scripts/update_lti.js

```diff
@@ -106,32 +108,32 @@ var updateToolsForCourse = function(course, index, callback) {
     return callback();
   }
 
-  CanvasAPI.getExternalToolsForCourse(course, function(err, courseTools) {
-    if (err) {
-      log.error({'err': err, 'course': course.id}, 'Failed to get external tool configurations for course');
+  CanvasAPI.getExternalToolsForCourse(course, function(apiErr, courseTools) {
+    if (apiErr) {
+      log.error({'err': apiErr, 'course': course.id}, 'Failed to get external tool configurations for course');
       results.error = results.error.concat(toolsToUpdate);
       return callback();
     }
 
-    async.eachOfSeries(toolsToUpdate, function(tool, index, done) {
+    async.eachOfSeries(toolsToUpdate, function(tool, toolIndex, done) {
```

Comment:
@paulkerschen I was careful with this renaming of `index`. Please give it a close look, too.